lookout/core/analyzer.py

```diff
@@ -68,18 +68,16 @@ class Analyzer:
     model_type = None  # type: Type[AnalyzerModel]
     name = None  # type: str
 
-    def __init__(self, model: AnalyzerModel, url: str, config: Mapping[str, Any]):
+    def __init__(self, model: AnalyzerModel, url: str):
```

Comment:
This breaks the API. We should still have config here even though we don't use it yet in this precise analyzer (in function name analyzer this is the only config we should use for example).